contracts/tests/sbtc-withdrawal.test.ts

```diff
@@ -150,12 +200,33 @@ describe("initiating a withdrawal request", () => {
     expect(
       txErr(
         withdrawal.initiateWithdrawalRequest({
-          amount: 100n,
+          amount: 1000n,
           recipient: newPoxAddr(6, new Uint8Array(20)),
           maxFee: 10n,
         }),
         alice
       ).value
     ).toEqual(errors.withdrawal.ERR_INVALID_ADDR_HASHBYTES);
   });
+
+  test("Cannot try and withdrawal less than or equal to the dust limit", () => {
```

Comment:
nit: To match the pattern above "test withdrawal amount of less than or equal to the dust limit is rejected"